app/views/blocks/opportunitiesQuestion.html

```diff
@@ -0,0 +1,18 @@
+<span class="help-block" translate>
+  Checking yes below will add you to a subscription list to receive monthly
+  updates on the latest Cru opportunities! You can unsubscribe at any time.
+</span>
+<div ng-repeat="choice in block.content.staticChoices" class="radio">
+  <label>
+    <input
+      type="radio"
+      show-errors
+      name="{{ block.id }}"
+      ng-required="block.required"
+      ng-model="answer.value"
+      ng-model-options="{ debounce: 300 }"
+      ng-value="choice"
+    />
+    <strong>{{ choice }}</strong></label
```

Comment:
Should choosing `Yes, via text` and `Yes, via email and text` open a input for the user to enter their telephone?

Reply:
They can add a Phone number profile question to get than info the same way they get their email.